src/pages/programs/index.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from "react";
+import { useState } from "react";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import Modal from "../../components/grid-aware/Modal";
+import ProgramBlock from "../../components/grid-aware/ProgramBlock";
+import ProgramsPageLargeParagraph from "../../components/grid-aware/ProgramsPageLargeParagraph";
+import ProgramsSubTitleBlock from "../../components/grid-aware/ProgramsSubTitleBlock";
+import Spacer from "../../components/grid-aware/Spacer";
+import Layout from "../../components/layout";
+import PartnershipSignupForm from "../../components/thirdparty/mailchimp/PartnershipSignupForm";
+import communityDevelopmentImg from "./community-development.jpg";
+import articleSpotlightImage from "./laura-barerra-vera.png";
```

Comment:
I think you have an older version of this image, where the image is cropped in a way where the Volunteer Spotlight box obscures her face. @candywang's very recent PR, https://github.com/ShelterTechSF/sheltertech.org/pull/267, updated the image such that Laura's face is further to the right, so could you pull in her updates and make sure you use this updated image?

Reply:
I just wanted to bump this comment, since it still looks like you're using the older image. You can tell the difference because in the older one, the black box containing the "Volunteer Spotlight" text overlaps her face, while the newer one doesn't.